### DISCUSSION TITLE
Making an Arch Linux Arm VM

### DISCUSSION THREAD

**Discussion by UdderNonsense** [utmapp/UTM#5880] [Linux]
I'd like to setup an Arch Linux Arm guest without the gallery link (see [https://mac.getutm.app/gallery/archlinux-arm]), but I am not experienced enough to do so through Arch Linux ARM's tarballs and Arch itself.

[https://archlinuxarm.org/about/downloads] at the bottom says I just have to provide my own Linux Kernel, which already befuddles me (I know what a kernel is). However, UTM requires a kernel *image*, and a root file system *image* and I am not sure if I have *images* or if I have just code. Is there a way I can safely make an image with these things?

I am doing this for the sake of it/to learn (or to spite myself, one or the other): I in no way actually need Arch Linux for anything. Ideally, I'd like tools/processes that are:

1. In more or less wide use
2. Verifiably safe
3. Easy to setup/remove from my machines
4. Available to run either on an M1 Mac or a Linux virtual machine on ARMv8 (i.e. use one of my other Linux vm's - I have a Focal Fossa one setup and backed up/cloned)

Ultimately if I could make my own ISO image that'd be cool (and maybe push it through Fedora Media writer)

**Comment by cynecx** [comment]
It's not really related to UTM but I've written down a short write-up how you can install an ArchLinux on an Virtualization.framework VM: https://github.com/cynecx/vmcli.
